src/Elmish.WPF/BindingInternal.fs

```diff
@@ -407,60 +401,46 @@ module internal BindingData =
         (mSet "set")
 
 
-  module OneWaySeqLazy =
+  module OneWaySeq =
 
     let mapMinorTypes
         (outMapA: 'a -> 'a0)
```

Comment:
Shouldn't `outMapA` and `inMapA` be renamed to `outMap` and `inMap` since they are no longer distinguished from other map functions?

Reply:
`outMapId` needs to be distinguished